setup.py

```diff
@@ -3,30 +3,30 @@
 from setuptools import setup
 
 setup(
-      name='twilio-tap-zendesk',
-      version='1.0.13',
-      description='Singer.io tap for extracting data from the Zendesk API',
-      author='Twilio',
-      url='https://github.com/twilio-labs/twilio-tap-zendesk',
-      classifiers=['Programming Language :: Python :: 3 :: Only'],
-      py_modules=['tap_zendesk'],
-      install_requires=[
-          'pipelinewise-singer-python==1.2.0',
-          'zenpy==2.0.25',
-          'requests==2.31.0'
-      ],
-      extras_require={
-          'dev': [
-              'ipdb',
-              'pylint',
-              'nose',
-              'nose-watch',
-          ]
-      },
-      entry_points='''
+    name="twilio-tap-zendesk",
+    version="1.0.17",
+    description="Singer.io tap for extracting data from the Zendesk API",
+    author="Twilio",
+    url="https://github.com/twilio-labs/twilio-tap-zendesk",
+    classifiers=["Programming Language :: Python :: 3 :: Only"],
+    py_modules=["tap_zendesk"],
+    install_requires=[
+        "pipelinewise-singer-python==1.2.0",
+        "zenpy==2.0.49",
+        "requests==2.31.0",
```

Comment:
While you're here: Do me a favor and bump this to 2.32.0, for [CVE reasons](https://weavegrid.atlassian.net/browse/EX-9839?atlOrigin=eyJpIjoiNWMwMjRiMTBhMGNiNGVhZWI2YTViYzBiNDMwNzcyY2EiLCJwIjoiaiJ9)

Reply:
@bdewilde sure thing. How do we do our CVE scans anyhow? I thought about looking that up when I cut this PR.

Reply:
I'm sure @susanshrack would be delighted to tell you about the automated CVE scanning system... :)

Reply:
Ha 😄 
This confluence page covers it at a high level: https://weavegrid.atlassian.net/wiki/spaces/EN/pages/2038497564/Vuln+Mgmt+Jira+Automation+and+Engineering+Response
TLDR is it's done via Trivy scans triggered via CircleCI jobs - both nightly scans & when merging to the main branch.